### DISCUSSION TITLE
Error No such file or directory

### DISCUSSION THREAD

**Discussion by victoralfonso920** [subosito/flutter-action#154] [General]
need help from the error 
build/app/outputs/apk/release/': No such file or directory

in run my workflow, i test from the rute 
build/app/outputs/fluter-apk/release/
build/app/outputs/flutter-apk

![Captura de Pantalla 2022-04-29 a la(s) 21 44 41](https://user-images.githubusercontent.com/7407933/166089301-a5f5d3e4-cd62-42b7-af27-ab1941f3102e.png)

**Comment by Saifallak** [comment]
try
```
            build/app/outputs/flutter-apk/app-release.apk
            build/app/outputs/bundle/release/app-release.aab
```
